CONTRIBUTING.md

```diff
@@ -5,56 +5,107 @@ When contributing to this repository, please describe the change you wish to mak
 Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
 ## Setting Up Development Environment
-Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools. 
-### Prerequisites
+
+Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools.
+
+### With Devbox
+
+The easiest way to setup and build devbox is with Devbox!
+
+* Install Devbox using the command below. If you don't have Nix installed, Devbox will automatically install it for you when you run a command.
+
+```bash
+curl -fsSL https://get.jetpack.io/devbox | bash
+```
+
+* Start a Devbox Shell for the project
+
+```bash
+devbox shell
```

Comment:
we don't really need this here. It's confusing users thinking they have to be in a devbox shell to run the build commands below

Reply:
Edited a little bit, but wanted to have that command available somewhere